service/src/test/java/bio/terra/drshub/controllers/DrsHubApiControllerTest.java

```diff
@@ -30,6 +30,7 @@
 import io.github.ga4gh.drs.model.AccessMethod;
 import io.github.ga4gh.drs.model.AccessMethod.TypeEnum;
 import io.github.ga4gh.drs.model.AccessURL;
+import io.github.ga4gh.drs.model.AllOfAccessMethodAccessUrl;
```

Comment:
I was wondering what this was / why we need it. ( my env can't find this -- I pulled the branch and reloaded Gradle and it comes up as unresolveable for me :/ )

Reply:
try a `~/.gradlew build`

Reply:
It's in order to fix a lint error in the older spec. If you want both a $ref and a description you have to wrap it in `AllOf` which unfortunately then gets propagated through the autogenerated code.

Reply:
gotcha, thanks!!